### DISCUSSION TITLE
Latest prerelease (20240616_1004) doesn't work for R36S panel 4

### DISCUSSION THREAD

**Discussion by jbiosca78** [351ELEC/351ELEC#1022] [Questions & Answers]
I have a R36S shipped this month, it is version V5 (panel 4)
When I install the latest prerelease, it has black screen. I wait for the start process (more than 10 minutes) and after boot it's still black.

I notice that the start process reboots machine (the screen backlight powers off and on), so I think that I wait enought time for it to finish. But nothing displays.

¿How AmberELEC detect new display for replace dtb? I think that the detection is missing my screen.
The dtbs after the init process have the same hashes as just after flash sd, so they don't change.

In some forums and discord chats there are people with same problem.

The sd card works fine with arkos patched for r36s panel 4 and rocknix after copying the dtbo file for panel 4.

I have ssh access and can run commands if needed.

The method of rocknix for use standard dtb and put only panel 4 overlay file seems nice. I'm going to try to use it for amberelec dtb.

PD: it would be nice if amberelec switchs led to red after finish first installation process, so R36 users can know when it finish and works or not. It's annoying to wait 10 minutes after each try, when I think that the process takes only 2 minutes or so.

**Comment by loganwishartcraig** [comment]
Same issue here.

**Reply by Dattebayooooo** [reply]
Can you share the latest release link? 2024

**Reply by RetroGFX** [reply]
https://github.com/AmberELEC/AmberELEC-prerelease/releases/
